koat: internal error, uncaught exception:
      Failure("Transition leading back to the initial location.")
      
